docs/Generators/index.rst

```diff
@@ -1,12 +1,18 @@
+.. _generator-index:
+
 Generators
 ============================
 
-Generator can be thought of as a generalized RTL design, written using a mix of meta-programming and standard RTL.
+A Generator can be thought of as a generalized RTL design, written using a mix of meta-programming and standard RTL.
 This type of meta-programming is enabled by the Chisel hardware description language (see :ref:`Chisel`).
 A standard RTL design is essentially just a single instance of a design coming from a generator.
 However, by using meta-programming and parameter systems, generators can allow for integration of complex hardware designs in automated ways.
 The following pages introduce the generators integrated with the Chipyard framework.
 
+Chipyard bundles the source code for the generators, under the ``generators`` directory.
+It builds them from source each time (although ``sbt`` will cache results if they have not changed),
```

Comment:
I would get rid of the reference to SBT and instead keep it general "although the build system will cache the results if they ..."